src/js/tabs/sensors.js

```diff
@@ -379,9 +379,15 @@ TABS.sensors.initialize = function (callback) {
 
                     samples_accel_i = addSampleToData(accel_data, samples_accel_i, FC.SENSOR_DATA.accelerometer);
                     drawGraph(accelHelpers, accel_data, samples_accel_i);
-                    raw_data_text_ements.x[1].text(FC.SENSOR_DATA.accelerometer[0].toFixed(2));
-                    raw_data_text_ements.y[1].text(FC.SENSOR_DATA.accelerometer[1].toFixed(2));
-                    raw_data_text_ements.z[1].text(FC.SENSOR_DATA.accelerometer[2].toFixed(2));
+                    const x = FC.SENSOR_DATA.accelerometer[0].toFixed(2);
+                    const y = FC.SENSOR_DATA.accelerometer[1].toFixed(2);
+                    const z = FC.SENSOR_DATA.accelerometer[2].toFixed(2);
+                    const pi = Math.PI;
+                    const rollACC = Math.round(Math.atan(y / (Math.sqrt(Math.pow(x, 2)) + (Math.pow(z, 2)))) * (180 / pi));
+                    const pitchACC = Math.round(Math.atan(x / (Math.sqrt(Math.pow(y, 2)) + (Math.pow(z, 2)))) * (180 / pi));
+                    raw_data_text_ements.x[1].text(`${x} ( ${rollACC} )`);
```

Comment:
As a suggestion, an screen capture will help to see the final result and help with the review. 

Without seeing the final result I suppose is better to remove the spaces with the parenthesis:
```suggestion
                    raw_data_text_ements.x[1].text(`${x} (${rollACC})`);
```

If we have enough space, that I doubt, we can directly add the units:
```suggestion
                    raw_data_text_ements.x[1].text(`${x}g (${rollACC} deg)`);
```
but we will need to move the string to the messages file.

What do you think?

Reply:
Updated. The spaces are for visual separating g-force and degrees for roll (X) and pitch (Y) access.

Reply:
I was talking about the spaces _inside_ the parenthesis, not between g-forge and degrees. My first suggestion, final result removing it:
![image](https://user-images.githubusercontent.com/2673520/95207761-7fa10c00-07e8-11eb-833a-476d35a6306d.png)

Reply:
Fixed!